lib/matplotlib/tests/test_style.py

```diff
@@ -46,8 +46,8 @@ def temp_style(style_name, settings=None):
 
 
 def test_deprecated_rc_warning_includes_filename():
-    SETTINGS = {'axes.color_cycle': 'ffffff'}
-    basename = 'color_cycle'
+    SETTINGS = {'foo': 'bar'}
+    basename = 'basename'
```

Comment:
Can't this test go now?

Reply:
Renamed as "test_invalid_rc_warning_includes_filename", which seems still a valid test.